Source/WebCore/page/ScriptTelemetryTypes.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "HTMLInputElement.h"
```

Comment:
We shouldn't have to include HTMLInputElement.
A forward declaration should be enough.